Update README.md (#1002)
returntocorp/sgrep-rules: README.md

| `master` | `returntocorp/semgrep:latest`   | [![semgrep-rules-test](https://github.com/returntocorp/semgrep-rules/workflows/semgrep/badge.svg?branch=master)](https://github.com/returntocorp/semgrep-rules/actions?query=workflow%3Asemgrep+branch%3Amaster) |
| `develop` | `returntocorp/semgrep:develop`  | [![semgrep-rules-test-develop](https://github.com/returntocorp/semgrep-rules/workflows/semgrep-develop/badge.svg)](https://github.com/returntocorp/semgrep-rules/actions?query=workflow%3Asemgrep-develop+branch%3Adevelop) |

Welcome! This repository is the “standard library” for [Semgrep](https://semgrep.dev) rules, but there are many more written by [r2c](https://r2c.dev) and other contributors available in the [Semgrep Registry](https://semgrep.dev/explore).

## How do I use these rules?

> We recommend starting with a pre-selected subset of rules that someone has already carefully tuned for precision. These rule packs, which may include rules not present in this repository, are available on the [Semgrep Registry](https://semgrep.dev/explore).

Reviewer: ```suggestion
We recommend starting with a pre-selected subset of rules that someone has already carefully tuned for precision. These rulesets, which may include rules not present in this repository, are available for free on the [Semgrep Registry](https://semgrep.dev/explore).
```

> The two most popular are:

Reviewer: ```suggestion
The two most popular rulesets are:
```

* `semgrep --config=p/ci` to find logic bugs, and high-confidence security vulnerabilities; recommended for CI

> * `semgrep --config=p/security-audit` to find security audit points; noisy, not recommended for CI

Reviewer 2: Suggestion: "recommended for manual scans"

> If you are determined to drink from the firehose, you can also pull directly from the registry by specifying `r/<foldername.subfolder.etc>`. So to run all the python flask rules you would specify:

Reviewer: ```suggestion
If you are determined to drink from the firehose, you can also pull directly from the registry by specifying `r/<foldername.subfolder.etc>`. So to run all the Python Flask rules specify:
```

`semgrep --config=r/python.flask`

> Semgrep releases [new versions at a very high frequency](https://github.com/returntocorp/semgrep/releases) and the rules in this repo are always updated to take advantage of new features or fix breaking changes on new releases. For that reason, we suggest that you avoid stale copies of registry rules.

Reviewer: ```suggestion
Semgrep releases [new versions very frequently](https://github.com/returntocorp/semgrep/releases) and the rules in this repo are always updated to take advantage of new features or fix breaking changes on new releases. For that reason, we suggest that you avoid stale copies of registry rules.
```

If you have more questions, please see the [FAQ section in the Semgrep docs](https://semgrep.dev/docs/faq).

## Running Rules in CI/Pre-Commit/Developer Workflow

> If you want run these rules rather than write them, [see the CI instructions on the Semgrep Registry](https://semgrep.dev/explore) (click through to any rule pack). That website is a convenient frontend for this repository.

Reviewer 3: @ievans Rather than casting the Registry as a convenient frontend to this repo, could we instead link to specific rulesets containing these rules? Going a little further, could we explain that we publish these rules to the Registry for convenient across-project distribution, as do others. Maybe we could link to Awesome Rulesets instructions.

Author: Did a major rewrite -- take another look!

If you have more questions, please see the [FAQ section in the Semgrep docs](https://semgrep.dev/docs/faq).

## Help

> Join [Slack](https://r2c.dev/slack) for the fastest answers to your questions! Or contact the team at [semgrep@r2c.dev](mailto:semgrep@r2c.dev).

Reviewer: ```suggestion
Join [Slack](https://r2c.dev/slack) for the fastest answers to your questions! Or contact the team at [support@r2c.dev](mailto:support@r2c.dev).
```
